client/src/pages/EditClassPage.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import Dropdown from "../components/Dropdown";
+import DropdownMulti from '../components/DropdownMulti';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrashAlt } from "@fortawesome/free-regular-svg-icons";
+
+const EditClassPage = () => {
+  return (
+    <div className="flex">
+      <div className="flex flex-col w-full  h-screen">
+        <h1 className="font-bold p-3 sticky top-0 z-50 bg-white text-lg border-b-2 fex">
+          Edit Class
+          <button
+            type="button"
+            className="p-2 ml-4 bg-regal-blue float-right text-white font-medium text-xs leading-tight uppercase rounded shadow-md hover:bg-blue-700 hover:shadow-lg focus:bg-blue-700 focus:shadow-lg focus:outline-none focus:ring-0 active:bg-blue-800 active:shadow-lg transition duration-150 ease-in-out"
+          >
+            <FontAwesomeIcon icon={faTrashAlt} className="mr-1" />
+            Delete
+          </button>
+        </h1>
+        <div className="relative h-full flex flex-col justify-center items-center overflow-hidden">
+          <div className="w-full p-6 m-auto bg-custom-gray rounded-md shadow-md lg:max-w-xl">
+            <form className="mt-1">
+              <div className="mb-4">
+                <label className="block text-sm font-semibold text-gray-800">
+                  Class Name
+                </label>
+                <input
+                  className="block w-full px-4 py-2 mt-2 bg-white border rounded-md focus:border-blue-500 focus:outline-blue-500  input"
+                />
+              </div>
+              <div className="mb-4">
+                <label className="block text-sm font-semibold text-gray-800">
+                  Department
+                </label>
+                <Dropdown type="department" label="fullname"/>
+              </div>
+              <div className="mb-4">
+                <label className="block text-sm font-semibold text-gray-800">
+                  Class Instructor
+                </label>
+                <Dropdown type="instructor" label="fullname"/>
+              </div>
+              <div className="mb-4">
+                <label className="block text-sm font-semibold text-gray-800">
+                  Class List
+                </label>
+                <DropdownMulti type="students" label="fullname"/>
+              </div>
+              <div className="mt-8">
+                <button className="w-full px-4 py-2 tracking-wide text-white transition-colors duration-200 transform bg-regal-blue rounded-md hover:bg-blue-900 focus:outline-none focus:bg-blue-900">
+                  Accept Changes
+                </button>
```

Comment:
You can directly apply our button component on this one to make it easier for the next co dev
`<SubmitButton handleSubmit={''} buttonDisabled={''} processing={''} buttonTitle={"Edit Class"} />`